src/services/AuthService.js

```diff
@@ -0,0 +1,25 @@
+const AuthService = {
+  getAccessToken: async function (cookies) {
+    let resultToken = '';
+    if (!cookies.token) {
+      const getTokenResult = await fetch(`/auth/get-token`, {
```

Comment:
Maybe it would be better to move this request to the api folder and here only call a method